### DISCUSSION TITLE
attributes Type

### DISCUSSION THREAD

**Discussion by vralle** [webdiscus/html-bundler-webpack-plugin#34] [Q&A] [answered]
Hi! I've been using [HTML Bundler Plugin for Webpack](https://github.com/webdiscus/html-bundler-webpack-plugin) for a while. This is a great tool for creating website prototypes. As new tasks were implemented, the webpack config grew and I have enabled strict type checking.

`loaderOptions.sources`:

![](https://github.com/webdiscus/html-bundler-webpack-plugin/assets/3823060/a6075e27-70d9-4bd8-8813-35cb032c1df9)

My config is working OK and I'm sure that `attributes` is an object. I believe this is a issue in the type export https://github.com/webdiscus/html-bundler-webpack-plugin/blob/f9c53f2d9438223753eab0639c9fa7b13f0a2c1f/types.d.ts#L215
I use html-bundler-webpack-plugin v2.14.3

**Comment by vralle** [comment]
README.md have same issue.
[sources](https://github.com/webdiscus/html-bundler-webpack-plugin/tree/f9c53f2d9438223753eab0639c9fa7b13f0a2c1f#sources)

**Reply by webdiscus** [reply, marked as answer]
@vralle 

thank you!
I have created new issue #35

**Reply by webdiscus** [reply]
@vralle 

please, for a new case/issue create new issue ticket. See #36 .
